src/materials/ConcreteThermalMoisture.C

```diff
@@ -144,6 +151,17 @@ ConcreteThermalMoisture::ConcreteThermalMoisture(const InputParameters & paramet
     _has_temperature(isCoupled("temperature")),
     _temp(_has_temperature ? coupledValue("temperature") : _zero)
 {
+  if (getParam<MooseEnum>("moisture_diffusivity_model") == 2 && _water_to_cement < 0.5)
+    mooseError("water_to_cement_ratio should be >= 0.5 to use Xi's model for moisture diffusivity");
+
+  if (getParam<std::string>("type") == "PorousMediaBase")
+    mooseWarning(
+        "PorousMediaBase is being replaced by ConcreteThermalMosture. Note that in "
+        "the new class, the behavior is changed for the thermal capacity for "
+        "ASCE-1992 and KODUR-2004 models, and the thermal conductivity for ASCE-1992, "
+        "KODUR-2004, and EUROCODE-2004 models. If the temperature is <= 20 C, the values "
+        "from the correlations at 20 C are used rather than user-supplied values. A similar "
+        "change was also made for the computation of N_wc for water-cement ratios <= 0.3.");
```

Comment:
Same here. The branch needs to be updated.

Reply:
corrected now.